### ISSUE TITLE
Local Variable is Initialized but Not Referenced

### ISSUE THREAD

**Issue by jwillikers** [boostorg/process#120]
I encounter this warning compiling an application that uses Boost process 1.72.0 with MSVC.
```cpp
include\boost\process\detail\windows\handles.hpp(37): warning C4189: 'cnt': local variable is initialized but not referenced
```

**Comment by jonesmz** [comment]
There is no 'cnt' variable in handls.hpp

**Comment by tt4g** [comment]
Already fixed in #97 but not released now.
